cid.go

```diff
@@ -79,6 +79,28 @@ const (
 	ZcashTx            = 0xc1
 )
 
+// Codecs maps the name of a codec to its type
+var Codecs = map[string]byte{
```

Comment:
It should probably be `uint64` as in https://github.com/ipfs/go-cid/blob/master/cid.go#L109